epilepsy12/models_folder/case.py

```diff
@@ -143,7 +143,7 @@ def age(self, today_date=date.today()):
     def save(self, *args, **kwargs) -> None:
         # calculate the index of multiple deprivation quintile if the postcode is present
         # Skips the calculation if the postcode is on the 'unknown' list
-        if self.postcode:
+        if self.postcode and not self.index_of_multiple_deprivation_quintile:
```

Comment:
This is a behaviour change: before we would look up this data on every save. But I don't think anything relied on that @eatyourpeas?

Reply:
This looks to be something I must have missed. Thanks for identifying. Yes, there is no need to make an API call if the IMD quintile has already been calculated and saved in the record.

Reply:
Cool let's fix them in separate PRs. Even though we're not doing continuous deployment on this project at the moment I think in general raising small focused PRs will be easier to manage both now and in the future.

I've raised #824 and #825 to track those.

@eatyourpeas just checking you're happy for me to merge this PR? :)